client/src/partials/finance/cashflow/cashflow.html

```diff
@@ -0,0 +1,239 @@
+<div class="flex-header static">
+  <div class="bhima-title">
+    <ol class="headercrumb">
+      <li class="static">{{ "TREE.FINANCE" | translate }}</li>
+      <li class="title">
+        {{ "CASHFLOW.TITLE" | translate }}
+        <b ng-show="ReportCtrl.state !== 'generate'" class="label label-default">
```

Comment:
We started using `ui-router` to get rid of this type of code.  See https://github.com/IMA-WorldHealth/bhima-2.X/issues/51 and https://github.com/IMA-WorldHealth/bhima-2.X/issues/206.

It seems like cashflow only has one `ui-router` state.  Could you propose how to make these `ReportCtrl.state`s into `ui-router` states?